web/sap_systems_api.go

```diff
@@ -0,0 +1,27 @@
+package web
+
+import (
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+	"github.com/trento-project/trento/web/services"
+)
+
+// ApiSAPSystemsHealthSummaryHandler godoc
+// @Summary Retrieve SAP Systems Health Summary
+// @Accept json
+// @Produce json
+// @Success 200 {object} HealthSummaryResponse
```

Comment:
Where is this `HealthSummaryResponse` defined? i cannot find it anywhere...

Reply:
updated with the right struct, thanks!